website/content/api-docs/acl/auth-methods.mdx

```diff
@@ -34,7 +34,9 @@ The table below shows this endpoint's support for
 | ---------------- | ----------------- | ------------- | ------------ |
 | `NO`             | `none`            | `none`        | `acl:write`  |
 
-### Payload Fields
+-> The corresponding CLI command is [`consul acl auth-method create`](/commands/acl/auth-method/create).
```

Comment:
After discussing with @karl-cardenas-coding and @trujillo-adam , we agree that we need to remove the info box treatment for these sentences (that gives them the blue background). Can you go through all these and remove the `-> ` at the front?

```suggestion
The corresponding CLI command is [`consul acl auth-method create`](/commands/acl/auth-method/create).
```

Reply:
I addressed my own feedback